File: Python/ByteBank/bytebank.py
from sistema.conta import ContaCorrente

# Objetos
carlos = ContaCorrente("Carlos Alberto de Nóbrega", 12321123221, 20000)
marcelo = ContaCorrente("Marcelo Souza Pinto", 2321423424, 10)

# Escopo
ContaCorrente.codigo_banco()
print()
carlos.transferir(200, marcelo)
marcelo.extrato()

File: PHP/ByteBank/ByteBank.php
<?php

require_once 'Sistema/Conta.php';

// Instância de Objetos
$junior = new Conta("Junior Matias Santos", 200);
$macela = new Conta("Macela Pernambucana", 1000);

// Escopo
echo $junior->depositar(200);
echo $macela->transferir(500, $junior);

// Saldo
echo PHP_EOL;
echo $junior->extrato();
echo $macela->extrato();